client/src/nl/sensorlab/videowall/animation/baseanimations/bouncypixels/BouncyPixel.java

```diff
@@ -0,0 +1,92 @@
+package nl.sensorlab.videowall.animation.baseanimations.bouncypixels;
+
+import java.util.ArrayList;
+
+import nl.sensorlab.videowall.animation.BaseAnimation;
+import processing.core.PGraphics;
+import processing.core.PVector;
+
+public class BouncyPixel {
+
+	BouncyPixelSystem parent;
+
+	public PVector position;
+	public PVector velocity;
+	public float diameter;
```

Comment:
If you work with radius here, it saves you a boatload of diameter / 2 in the update functions. Makes for better performance.